source/DataElement/AMDSBufferGroupInfo.h

```diff
@@ -49,18 +52,20 @@ class AMDSBufferGroupInfo
 	inline quint64 spanSize() const;
 
 	/// Set the name of the bufferGroupInfo
-	void setName(const QString &name) { name_ = name; }
+	inline void setName(const QString &name) { name_ = name; }
 	/// Set the description of the bufferGroupInfo
-	void setDescription(const QString &description) { description_ = description; }
+	inline void setDescription(const QString &description) { description_ = description; }
 	/// Set the units of the bufferGroupInfo
-	void setUnits(const QString &units) { units_ = units; }
+	inline void setUnits(const QString &units) { units_ = units; }
+	/// Set the dataType of the bufferGroupInfo
+	inline void setDataType(const AMDSDataTypeDefinitions::DataType dataType) { dataType_ = dataType; }
```

Comment:
Typically, enums are not `const` unless you were actually wanting a `const int`.